dev/src/bundle.ts

```diff
@@ -113,8 +118,12 @@ export class BundleBuilder {
     }
   }
 
-  private lengthPrefixedBuffer(payload: string): Buffer {
-    const buffer = Buffer.from(payload, 'utf-8');
+  // Converts a IBundleElement to a Buffer whose content is the length prefixed JSON representation
```

Comment:
For method comments, prefer JSDoc:

```
/**
 * Converts a IBundleElement...
 * @private
 */
```

Note that you have to add `@private` to make sure this doesn't show up in our public docs. JSDoc doesn't know that this method is TypeScript private.